project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,492 @@
+        apply_logic:
+          and:
+            - Geometry Foundation Type|Vented Crawlspace
+            - or: # Ensure that the dwelling unit being modeled is either (1) not in a multifamily building, or (2) on the bottom floor of a multifamily building and therefore above the crawlspace.
+                - Geometry Building Level MF|None
+                - Geometry Building Level MF|Bottom
+        costs:
+          - value: 1.083 # $/ft2 (0.84* $/ft2 * 1.289 inflation) https://remdb.nrel.gov/measures.php?gId=14&ctId=334&scId=5169&acId=6565)
+            multiplier: Floor Area, Foundation (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      #insulate finished attics and cathedral ceilings to R-30
+  - upgrade_name: Advanced Enclosure
+    #Up
+    options:
+      #Light Enclosure Measures
+      - *attic_insulation_IECC_CZ1A
+      - *attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+      - *attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
+      - *infiltration_30pct_reduction
+      #Intermediate Enclosure Measures
+      - *ducts_category1
+      - *ducts_category2
+      - *ducts_category3
+      - *ducts_category4
+      #- *ducts_category5
+      #- *ducts_category6
+      - *ducts_category7
+      - *Drill-and-fill
+      - *foundation_wall_insulation
+      - *foundation_wall_insulation_finished_basement
+      - *rim_joist_insulation
+      - *seal_vented_crawlspaces
+      #EnergyStar Windows
+      - *EnergyStar_window_IECC_CZ1A-2A-2B
+      - *EnergyStar_window_IECC_CZ3A-3B-3C
+      - *EnergyStar_window_IECC_CZ4A-4B-4C
+      - *EnergyStar_window_IECC_CZ5A-5B-6A-6B-7A-7B
+      #1” (TBD EPS or XPS) Exterior wall insulation for dwelling units with wall insulation of less than R-19  
+      - &exterior_wall_insulation
+        option: Insulation Wall|ICF, 2-in EPS, 4-in Concrete, 2-in EPS #(TBD)
+        apply_logic:
+          - or:
+            - Insulation Wall|Wood Stud, Uninsulated
+            - Insulation Wall|Wood Stud, R-7
+            - Insulation Wall|Wood Stud, R-11
+            - Insulation Wall|Wood Stud, R-13
+            - Insulation Wall|Wood Stud, R-15
+            - Insulation Wall|Wood Stud, Uninsulated, R-5 Sheathing
+            - Insulation Wall|Wood Stud, R-7, R-5 Sheathing
+            - Insulation Wall|Wood Stud, R-11, R-5 Sheathing
+            - Insulation Wall|Wood Stud, R-13, R-5 Sheathing
+            - Insulation Wall|Wood Stud, R-15, R-5 Sheathing
+            - Insulation Wall|Steel Stud, Uninsulated
+            - Insulation Wall|Steel Stud, R-13
+        costs:
+          - value: 0.0443 # TBD
+            multiplier: Floor Area, Attic * Insulation Increase (ft^2 * Delta R-value) # TBD
+        lifetime: 60
+      - &infiltration_IECC2021_with_ERV_added
+        option: Infiltration|3 ACH50
+        apply_logic:
```

Comment:
Can add anchor here to reuse logic below

Reply:
I did not use anchor before. Could you give me a link of yml file that use anchor? I can refer to that yml file.

Reply:
Done